active_directory/metadata.csv

```diff
@@ -1,31 +1,31 @@
-active_directory.dra.inbound.bytes.total,gauge,,byte,,The total number of bytes (per second) received through replication. It is the sum of the number of bytes of uncompressed data (never compressed) and compressed data (after compression).,-1,active_directory,ad dra inbound total
-active_directory.dra.inbound.objects.remaining,gauge,,object,,The number of objects remaining until the full synchronization process is completed.,-1,active_directory,ad dra sync full objs
-active_directory.dra.inbound.objects.persec,gauge,,object,second,The number of objects received (per second) through inbound replication from replication partners.,-1,active_directory,ad dra inbound objs
-active_directory.dra.inbound.objects.applied_persec,gauge,,object,second,"The number of objects received (per second) from replication partners and applied by the local directory service. This counter excludes changes that are received but not applied (for example, when the update is already made). This counter indicates how many replication updates are occurring on the server as a result of changes generated on other servers.",-1,active_directory,ad dra inbound objs applied
-active_directory.dra.inbound.objects.filtered_persec,gauge,,object,second,The number of objects received (per second) from replication partners that contained no updates that needed to be applied.,-1,active_directory,ad dra inbound objs filtered
-active_directory.dra.inbound.objects.remaining_in_packet,gauge,,object,second,"The number of object updates received in the current directory replication update packet that have not yet been applied to the local server. This counter tells you whether the monitored server is receiving changes, but is taking a long time applying them to the database.",-1,active_directory,ad dra remaining updates
-active_directory.dra.inbound.properties.applied_persec,gauge,,,second,The number of changes (per second) to object properties that are applied through inbound replication as a result of reconciliation logic.,-1,active_directory,ad dra properties applied
-active_directory.dra.inbound.properties.filtered_persec,gauge,,,second,The number of changes (per second) to object properties received during the replication that are already made.,-1,active_directory,ad dra properties filtered
-active_directory.dra.inbound.properties.total_persec,gauge,,,second,The total number of changes (per second) to object properties received from replication partners.,-1,active_directory,ad dra properties total
-active_directory.dra.inbound.values.dns_persec,gauge,,,second,"The number of values of object properties received (per second) from replication partners in which the values are for object properties that belong to distinguished names. This number includes objects that reference other objects. A high number from this counter might explain why inbound changes are slow to be applied to the database.",-1,active_directory,ad dra inbound dns
```

Comment:
Why were the quotes removed?

Reply:
So when my script opened and resaved the metadata.csv files, it appears that wrapping quotes that are not necessary (no `,` or `"` in the string) were removed from the value. Think this would cause any issues?

Reply:
And here's the script I used in case you want to take a look - https://gist.github.com/vatteh/3160c9552e511f00b4f4d69a4e100787

Reply:
I don't think it will cause issues, but I think it would be nice if the script made as little changes as possible, ie only adding the new column